examples/top_level.mk

```diff
@@ -33,23 +33,25 @@
 # ------------------------------------------------------------------------------
 # Author: A. R. Porter, STFC Daresbury Laboratory
 
-# Provides support for 'all', 'compile', 'clean', 'allclean' and
+# Provides support for 'all', 'compile', 'clean', 'allclean', 'notebooks' and
```

Comment:
For some reason I like clean and allclean at the end of the list. Feel free to ignore if you disagree.

Reply:
Done.